src/ukify/ukify.py

```diff
@@ -65,6 +65,9 @@
 }
 EFI_ARCHES: list[str] = sum(EFI_ARCH_MAP.values(), [])
 
+# Default and the first configuration file to read, will not complain if it does not exist.
+DEFAULT_CONFIG_FILE = '/etc/systemd/ukify.conf'
```

Comment:
The general expectation for systemd tools is to look in a bunch of places, and pick the file with the highest priority: `{/usr/lib/systemd,/usr/local/lib,/etc/systemd/,/run/systemd}/ukify.conf`.